includes/admin/tools/views/html-admin-page-system-info.php

```diff
@@ -489,6 +489,11 @@
 			<td><?php echo esc_html( ! empty( $default_gateway ) ? $default_gateway : '&ndash;' ); ?></td>
 		</tr>
 		<tr>
+			<td data-export-label="PayPal IPN Notifications"><?php _e( 'PayPal IPN Notifications', 'give' ); ?>:</td>
+			<td class="help"><span class="give-tooltip give-icon give-icon-question" data-tooltip="<?php echo esc_attr( __( 'Whether admin has enabled PayPal IPN notifications or not.', 'give' ) ); ?>"></span></td>
+			<td><?php echo 'enabled' === give_get_option( 'paypal_verification' ) ? __( 'Enabled', 'give' ) : __( 'Disabled', 'give' ); ?></td>
```

Comment:
This isn't correct because this is simply checking whether the `paypal_verification` if active, _not_ whether an IPN has actually been received or not. 

Please re-read over my scenarios comment on #1787 where I describe how this should work. If you have questions, we can hop on a call. Basically, you need to flag an option once an IPN _has been_ received.